platform/src/test/java/platformfeatures/PF_Footer.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Validate footer links
+@FooterPage
```

Comment:
Name is exclusive which is good.
Adding feature file name to tag would be better